storage/sealer/sealtasks/task.go

```diff
@@ -48,11 +50,12 @@ var order = map[TaskType]int{
 	TTCommit1:             2,
 	TTUnseal:              1,
 
-	TTFetch:    -1,
-	TTFinalize: -2,
+	TTFetch:          -1,
+	TTDownloadSector: -2,
```

Comment:
why is DownloadSector a higher priority than Commit1/2? Don't the latter have some time limits or am I mistaken?

Reply:
Downloading sector data in most cases will have similar time limits.

Note that those priorities will only apply when you have "universal" workers, which can execute all task types, which likely isn't very common.

When you do have universal workers, the point of the priority is to first work on finishing sectors in later stages of the sealing process, and move them quickly to the next sealing step - all in order to minimize required caches/scratch space.

Reply:
To your point, isn't commit1/2 the later stage of the sealing process over download sector? Or are those not considered part of the "sealing"?

Reply:
C1/C2 are part of the sealing pipeline, but they aren't small tasks which are moving data from/to different caches; Fetch/Download are also a bit awkward to assign priority here because they can run at different points in the sealing pipeline

This is possibly worth revisiting when we look at scheduler improvements.